Source/WebKit/Shared/ios/NativeWebTouchEventIOS.mm

```diff
@@ -118,10 +118,18 @@ static CGFloat radiusForTouchPoint(const WKTouchPoint& touchPoint)
     });
 }
 
+Vector<WebTouchEvent> NativeWebTouchEvent::extractCoalescedWebTouchEvents(const WKTouchEvent& event, UIKeyModifierFlags flags)
+{
+    return event.coalescedEvents.map([&](auto& event) -> WebTouchEvent {
+        return NativeWebTouchEvent { event, flags };
```

Comment:
Nit: since the return type hint exists, can you get away with `return { event, flags };`?

Reply:
nope, since it needs to go through the NativeWebTouchEvent constructor